TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8313,6 +8314,9 @@ lockTableStatement: SQL_LOCK SQL_TABLE tableOrViewOrCorrelationName (SQL_PARTITI
 share: ({ string.Equals(CurrentToken.Text, "SHARE", System.StringComparison.OrdinalIgnoreCase) }? KeywordSHARE=UserDefinedWord);
 exclusive: ({ string.Equals(CurrentToken.Text, "EXCLUSIVE", System.StringComparison.OrdinalIgnoreCase) }? KeywordEXCLUSIVE=UserDefinedWord);
 sql_mode: ({ string.Equals(CurrentToken.Text, "MODE", System.StringComparison.OrdinalIgnoreCase) }? KeywordMODE=UserDefinedWord);
+
+dropTableStatement: SQL_DROP SQL_TABLE tableOrAliasName;
```

Comment:
No,  correlation name is not allowed . I juste used ```tableOrViewOrCorrelationName``` to represent a qualified name.
It can represent a table or a view.

Reply:
Yes, we use the same class but the 2 methods  are different. In one we may have a 'correlation name' and no ambiguity. In the other we may have an 'alias' and an ambiguity. So I think we have to keep the two distinct methods.

Reply:
Ok, see my other remarks, we'll use a SymbolReference directly.